config/config/sc-config.yaml

```diff
@@ -899,6 +899,18 @@ opensearch:
     nodeSelector: {}
     tolerations: []
 
+  promIndexAlerts:
+    - pattern: kubeaudit-default
+      alertSizeMB: 4000
+    - pattern: authlog-default
+      alertSizeMB: 0.2
+    - pattern: other-default
+      alertSizeMB: 400
+    - pattern: security-auditlog
+      alertSizeMB: 16
+    - pattern: kubernetes-default
+      alertSizeMB: 4000
```

Comment:
Great stuff! Though I'd want to have this and perhaps `kubeaudit` set to say `5500` instead to be `500` over [the limit set as default for production environments](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/flavors/prod/sc-config.yaml#L41).

Reply:
thanks you. I missed that.